app/qml/main.qml

```diff
@@ -183,9 +185,10 @@ ApplicationWindow {
 
     OpenProjectPanel {
         id: openProjectPanel
-        height: window.height
-        width: QgsQuick.Utils.dp * 600
-        edge: Qt.LeftEdge
+        height: window.height - mainPanel.height
```

Comment:
I think it should take the whole window height. Different pages will have different content of their toolbar, so I would suggest that each page has its own toolbar (otherwise having one toolbar shared for all pages will quickly get difficult)